cobigen-cli/src/test/java/com/devonfw/cobigen/cli/commandtests/GenerateCommandTest.java

```diff
@@ -103,24 +106,36 @@ public void generateFromOpenApiTest() throws IOException {
         // Act
         CobiGenCLI.main(args);
 
-        // Assert
-        Path rootPath = outputRootFile.toPath();
-
-        File generatedFiles = rootPath.resolve("src/main/java/com/devonfw/angular/test/salemanagement").toFile();
-        assertTrue(generatedFiles.exists());
-
-        generatedFiles = rootPath.resolve("src/main/java/com/devonfw/angular/test/shopmanagement").toFile();
-        assertTrue(generatedFiles.exists());
-
-        generatedFiles = new File(testFileRootPath + "/devon4ng-ionic-application-template");
-        assertTrue(generatedFiles.exists());
-        FileUtils.deleteDirectory(generatedFiles);
-
-        // If you want to remove the generated files
-        FileUtils.deleteDirectory(rootPath.resolve("src").toFile());
-        FileUtils.deleteDirectory(rootPath.resolve("docs").toFile());
-        FileUtils.deleteDirectory(outputRootFile.getParentFile().toPath().resolve("api").toFile());
+     // Assert
+     		Path rootPath = outputRootFile.toPath();
+     		File generatedFiles = rootPath.resolve("src/main/java/com/devonfw/angular/test/salemanagement").toFile();
+     		geneatedList.add(generatedFiles);
+     		generatedFiles = rootPath.resolve("src/main/java/com/devonfw/angular/test/shopmanagement").toFile();
+     		geneatedList.add(generatedFiles);
+     		generatedFiles = new File(testFileRootPath + "/devon4ng-ionic-application-template");
+     		geneatedList.add(generatedFiles);
+     		geneatedList.add(rootPath.resolve("src").toFile());
+     		geneatedList.add(rootPath.resolve("docs").toFile());
+     		geneatedList.add(outputRootFile.getParentFile().toPath().resolve("api").toFile());
+     		GenerateCommandTest.isDeleteGeneratedFiles(geneatedList);
+     		geneatedList.clear();
     }
+    /**
+     * This method is check whether generated file is exist or not  
+     */
+	private static Boolean isDeleteGeneratedFiles(ArrayList<File> generateFiles) {
+		boolean deletedFiles = false;
+
+		for (File generatedFile : generateFiles) {
+			assertTrue(generatedFile.exists());
+			try {
+				FileUtils.deleteDirectory(generatedFile);
+			} catch (IOException e) {
+				continue;
+			}
+		}
+		return deletedFiles;
```

Comment:
This return variable is never used. Please change the method type to `void` and change the name to 'deleteGeneratedFiles'